modules/oxide-components/vitest.setup.ts

```diff
@@ -0,0 +1,2 @@
+// @ts-expect-error the bundler handles this just fine but tsc is not happy with it
+import '@tinymce/oxide/build/skins/ui/default/skin.css';
```

Comment:
isn't there any other way to handle this import? Or adjust it in a way that makes TS happy?

Maybe just make it JS 🤷‍♂️

Reply:
Indeed I just made it JS